### ISSUE TITLE
Feature Request - HA new Auth

### ISSUE THREAD

**Issue by emouawad** [Crewski/HANotify#6]
with HA moving out from legacy api-password i would be more confident to use this addon if this something planned or in the works - thanks - appreciate the hard works put into it anyway!

**Comment by kvvoff** [comment]
Yes, it would be great to use an access token, and not an outdated api-password.

**Comment by gcorgnet** [comment]
Yeah, I don;t think HA even supports API passwords anymore. Is there a way this could be updated to use a token please?
